modules/installation-mirror-repository.adoc

```diff
@@ -35,7 +67,11 @@ $ export LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
 $ export RELEASE_NAME="ocp-release" <6>
 ----
 <1> For `<release_version>`, specify the version number of {product-title} to
-install, such as `4.2.0`.
+install, such as `4.2.1`.
+ifdef::update[]
+When you update {product-title}, you must specify a version number that is
+higher than the version that is installed.
```

Comment:
I think we might support rollbacks from 4.y.z to 4.y.(z-1), although you should probably get someone with more authority to confirm that.  Also, I'm not sure all increases are supported.  For example, we no longer recommend upgrading to 4.1.12, so even though 4.1.11->4.1.12 is an increase, it may be an unsupported upgrade.

Reply:
I made a new PR based on the discussion on Slack about downgrades: https://github.com/openshift/openshift-docs/pull/16914
I'm moving this info to the previous step and saying "valid version." (I don't see 4.1.12 in the product link, so if they actually follow that step, they should pick something usable.)